src/test/java/com/direwolf20/buildinggadgets/common/building/blockprovider/OriginWrapperTest.java

```diff
@@ -0,0 +1,30 @@
+package com.direwolf20.buildinggadgets.common.building.blockprovider;
+
+import com.direwolf20.buildinggadgets.common.building.placement.IBlockProvider;
+import net.minecraft.util.math.BlockPos;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class OriginWrapperTest {
+
+    @Test
+    public void wrapperShouldTranslateParameterByAddingMethodGetTranslation() {
```

Comment:
Please test more than one Translation. Saving and Writing NBT should also be verified some where (Not here)